gateway-home/modules/ROOT/pages/gateway-overview.adoc

```diff
@@ -6,167 +6,48 @@ endif::[]
+Anypoint Flex Gateway is an ultrafast lightweight API gateway designed to manage and secure APIs running anywhere.
+Built to seamlessly integrate with DevOps and CI/CD workflows, Anypoint Flex Gateway delivers the
+performance required for the most demanding applications and microservices while providing enterprise
+security and manageability across any environment.
 
 For more information, see xref:gateway::flex-gateway-getting-started.adoc[Getting Started with Flex Gateway].
 
 == Anypoint Mule Gateway
 
-While Flex Gateway is a high-performance Envoy-based gateway with a small footprint, MuleSoft also provides a simple Mule Gateway for Mule Applications.
-
-The Mule Runtime, based on Java Spring, includes an embedded Mule Gateway. By using this gateway, you can, for example:
-
-* Apply a basic authentication policy on top of a Mule application.
-* Enrich an incoming or outgoing message.
-* Add any other complex capability to an API without having to write any code.
+Mule Runtime includes an embedded Mule Gateway. Using this gateway, any user can, for example, apply a basic authentication policy on top of a Mule application, enrich an incoming/outgoing message, or add any other complex capability to an API without having to write any code.
 
-Mule Gateway protects a single Mule API. The key advantage is that for application developers building Mule applications, basic endpoint protection becomes trivial.
+Overall, Mule Gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns. You can leverage the governance capabilities of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
 
-You can also configure Mule Runtime in Anypoint Runtime Manager as a CloudHub Proxy application, protecting multiple backends. However, in this case the gateway cannot be easily scaled to support high-availability and high-performance applications. Further, Mule Runtime consumes more computes or vCores than the smaller-footprint Flex Gateway.
+If you want to apply policies and generate analytics info, you must use either:
 
-Building custom policies on the Mule gateway is akin to building an application with Java using the Mule DSL. Building a custom policy in Flex Gateway is based on Envoy-provided Rust WASM SDKs. A policy written for Mule Gateway cannot be reused in Flex Gateway and vice versa, as the underlying architectures are fundamentally different.
+* An API defined in a Mule application.
+* A Mule application that proxies your existing backend implementation.
 
 For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[Mule Gateway Capabilities].
```

Comment:
The topic this links to is an overview for Mule Gateway. I suggest changing the title to `Mule Gateway Overview` to be consistent with the other topics.

Reply:
Fixed